packages/roosterjs-content-model-api/test/publicApi/table/applyTableBorderFormatTest.ts

```diff
@@ -26,7 +26,7 @@ describe('applyTableBorderFormat', () => {
         format?: ContentModelTableCell['format']
     ) {
         // Create a table with all cells selected except the first and last row and column
-        const table = createTable(rows);
+        const table: ContentModelTable = createTable(rows);
```

Comment:
Curious, any special reason to explicitly declare the type here?
(It is ok to add type declaration, just want to understand if anything is broken without this type here)

Reply:
Just stronger typing.